python/sparkdl/transformers/tf_tensor.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2017 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import logging
+import numpy as np
+import tensorflow as tf
+import tensorframes as tfs
+
+from pyspark.ml import Transformer
+from pyspark.ml.param import Param, Params
+from pyspark.sql.functions import udf
+
+from sparkdl.graph.builder import IsolatedSession
+import sparkdl.graph.utils as tfx
+from sparkdl.transformers.param import (
+    keyword_only, HasInputCol, HasOutputCol, SparkDLTypeConverters,
+    HasTFGraph, HasInputTensor, HasOutputTensor)
+
+__all__ = ['TFOneDimTensorTransformer']
+
+logger = logging.getLogger('sparkdl')
+    
+def _tranform_with_graph_function(gfn, df, input_col, output_col):
+    """ Apply a transform with a graph function
+    """
+    analyzed_df = tfs.analyze(df)
+
+    with IsolatedSession() as issn:
+        feeds, fetches = issn.importGraphFunction(gfn, prefix='')
+        assert len(feeds) == 1, 'only support single input TF model'
+        assert len(fetches) == 1, 'only support single output TF model'
+
+        # Change the column name for TensorFrames
```

Comment:
it's not much code -- why not just put it in the transformer and skip having to convert from/to GraphFunction? also can keep using the IsolatedSession there then.